Source/Csla.AspNetCore/Blazor/ApplicationContextManagerInMemory.cs

```diff
@@ -55,19 +56,18 @@ public ApplicationContextManagerInMemory(AuthenticationStateProvider authenticat
       InitializeUser();
     }
 
-    private void InitializeUser()
+    private async Task InitializeUser()
```

Comment:
Please apply the same comments here as in the other Manager.

Reply:
Done...and done